### PR TITLE
I brek thing s

### PR DESCRIPTION
**Pull request by practicalRope**
i unbroke...

### CODE REVIEW

File: app/src/routes/api/questions.js [hunk 1]
```diff
@@ -2,13 +2,14 @@ import fetch from "node-fetch";
 
 export async function get(req, res) {
     try {
-        const url = `${process.env.API_URL}/api/v1/questions?multichoice=6&dropdown=6`;
+        const url = `${process.env.API_URL}api/v1/questions?multichoice=6&dropdown=6`;
```
Comment (tagptroll1) [inline app/src/routes/api/questions.js]:
```suggestion
        const url = `${process.env.API_URL}/api/v1/questions?multichoice=6&dropdown=6`;
```
Please has `/` thx

File: app/src/routes/api/questions.js [hunk 2]
```diff
@@ -2,13 +2,14 @@ import fetch from "node-fetch";
...
 
         const resp = await fetch(url);
         const json = await resp.json();
 
         res.setHeader("Content-Type", "application/json");
         res.end(JSON.stringify(json));
+
```
Comment (tagptroll1) [inline app/src/routes/api/questions.js]:
```suggestion
```

File: app/src/routes/index.svelte [hunk 3]
```diff
@@ -1,58 +1,195 @@
 <script>
-  import Logo from "../components/LogoUIB.svelte";
+  import { goto } from "@sapper/app"
+  import { onMount,onDestroy } from "svelte";
+  import { slide, fade } from "svelte/transition"
```
Comment (tagptroll1) [inline app/src/routes/index.svelte]:
```suggestion
  import { slide, fade } from "svelte/transition";
```

File: app/src/routes/index.svelte [hunk 4]
```diff
@@ -1,58 +1,195 @@
...
+				const r = 64 + (128 * x / canvas.width) + (64 * Math.sin(t / 2500));
+				const g = 64 + (128 * y / canvas.height) + (64 * Math.cos(t / 2500));
+				const b = 128;
+
+				imageData.data[p + 0] = r * .9;
+				imageData.data[p + 1] = g * .4;
+				imageData.data[p + 2] = b * .7;
+				imageData.data[p + 3] = 255;
+			}
+
```
Comment (tagptroll1) [inline app/src/routes/index.svelte]: Kan du fikse tabsene her pls?

File: app/src/routes/index.svelte [hunk 5]
```diff
@@ -1,58 +1,195 @@
...
+  let valid = true;
+  let placeholder = "ex; rob404";
+  let numberOfAttempts = 0;
+  let wrongInput;
+
+  const alternatives = [
+    'joe432',
+    'tom233',
+    'aar444',
+    'jef007'
```
Comment (tagptroll1) [inline app/src/routes/index.svelte]:
```suggestion
    'jef007',
```

File: app/src/routes/index.svelte [hunk 6]
```diff
@@ -1,58 +1,195 @@
...
+    const key = event.key;
+    const code = event.keyCode;
+    if(key === "Enter" || code === 13){
+      start();
+      event.preventDefault();
+    }
+  }
+
+  function start(){
+    valid = input.checkValidity()
```
Comment (tagptroll1) [inline app/src/routes/index.svelte]:
```suggestion
    valid = input.checkValidity();
```

File: app/src/routes/index.svelte [hunk 7]
```diff
@@ -1,58 +1,195 @@
...
+  :global(.err){
+    animation: shake 0.4s ease;
+  }
+  @keyframes shake{
+			0% { transform: translate(10px); }
+			20% { transform: translate(-10px); }
+			40% { transform: translate(5px); }
+			60% { transform: translate(-5px); }
+			80% { transform: translate(3px); }
+			100% { transform: translate(0px); }
```
Comment (tagptroll1) [inline app/src/routes/index.svelte]: tabs, what.

File: app/src/routes/index.svelte [hunk 8]
```diff
@@ -1,58 +1,195 @@
...
+  button:focus,
+  button::-moz-focus-inner{
+    outline: none;
+    border:none;
   }
-  a {
-    text-decoration: none;
+
+
+
```
Comment (tagptroll1) [inline app/src/routes/index.svelte]:
```suggestion
```

File: app/src/routes/index.svelte [hunk 9]
```diff
@@ -1,58 +1,195 @@
...
+  button:focus,
+  button::-moz-focus-inner{
+    outline: none;
+    border:none;
   }
-  a {
-    text-decoration: none;
+
+
```
Comment (tagptroll1) [inline app/src/routes/index.svelte]:
```suggestion
```

File: app/src/routes/index.svelte [hunk 10]
```diff
@@ -61,71 +198,79 @@
...
     position: fixed;
     z-index: -1;
     display: flex;
     justify-content: center;
     align-items: center;
     overflow: hidden;
-    width: 100vw;
+
+    width: 100%;
+		height: 100%;
```
Comment (tagptroll1) [inline app/src/routes/index.svelte]:
```suggestion
    height: 100%;
```

File: app/src/routes/index.svelte (hunk 4 above)
Comment (practicalRope) [reply app/src/routes/index.svelte]: did

File: app/src/routes/index.svelte (hunk 7 above)
Comment (practicalRope) [reply app/src/routes/index.svelte]: idk

**Pull request merged**